### DISCUSSION TITLE
Mentored projects: Pedagogical Interactive Notebooks for Algorithms

### DISCUSSION THREAD

**Discussion by inomag** [networkx/networkx#5379] [General]
While skimming through the reference, I came across the Mentored projects section. I want to contribute to the interactive notebooks work as most of the algorithms are used extensively in our college projects.
Is there any restrictions on which algorithms the notebooks are to be made?

**Comment by MridulS** [comment]
Our current focus of https://github.com/networkx/nx-guides/ (this is where we host our notebooks) is to have pedagogical notebooks for algorithms already implemented in networkx. To get an overview of the algorithms you can go through our  documentation available at https://networkx.org/documentation/latest/reference/algorithms/index.html.

**Reply by inomag** [reply]
Hey,  I am a 3rd year Computer Science student  currently studying in National Institute of Technology, Silchar. I am familiar with Python and Networkx library and hoping to participate in GSOC 2022 by contributing to Networkx. Out of both projects in Networkx for Gsoc, I am interested in the "Pedagogical Interactive Notebooks for Algorithms Implemented in NetworkX" project.

There are several algorithms in Shortest Path, Lowest Common Ancestor, Distance Measures, Communities and a few others for which I could make notebooks using sample data and visualize them in a comprehensive way. I have used these algorithms for our Social Network Analysis projects.

This is my first time participating in GSOC and any guidance on how should I approach is greatly appreciated. Thanks
